docs/docs-book/src/contributing-to-coffee.md

```diff
@@ -0,0 +1,142 @@
+# Contributing to Coffee
+
+## Table of Content
+
+- Introduction
+- Code Style
+- Commit Style
+- How to make the release
+
+## Introduction
+
+Welcome to the HACKING guide and let's peek into how a day in the life of a "core lightning Rust Framework" maintainer looks like.
```

Comment:
```suggestion
Welcome to the HACKING guide and let's peek into how a day in the life of a "Coffe plugin manager" maintainer looks like.
```

Please pay attention and not just copy and paste!

Reply:
:)